Calibration/HcalCalibAlgos/plugins/HcalIsoTrkAnalyzer.cc

```diff
@@ -113,49 +122,59 @@ class HcalIsoTrkAnalyzer : public edm::EDAnalyzer {
   std::vector<unsigned int> *t_DetIds;
   std::vector<double>       *t_HitEnergies, pbin;
   std::vector<bool>         *t_trgbits; 
-  std::vector<std::string>   trgnames;
+  int                        t_Tracks, t_TracksProp, t_TracksSaved;
+  std::vector<int>          *t_ietaAll, *t_ietaGood;
 };
 
 HcalIsoTrkAnalyzer::HcalIsoTrkAnalyzer(const edm::ParameterSet& iConfig) : 
-  changed(false), nRun(0), nAll(0), nGood(0) {
+  nRun_(0) {
+
+  usesResource("TFileService");
```

Comment:
Following the suggestion in https://github.com/cms-sw/cmssw/pull/10176#discussion_r41967044 using the explicit name of the resource.